Class1.cs

```diff
@@ -13,10 +13,13 @@ public class IncreaseHuntressRange : BaseUnityPlugin
     {
         private static ConfigWrapper<bool> godMode;
         private static ConfigWrapper<float> scaleFactor;
+        private static ConfigWrapper<float> baseValue;
+
         public void Awake()
         {
             godMode = Config.Wrap<bool>("Huntress", "GodMode", "Set the range to be so massive that you can attack any enemy that is visible", false);
             scaleFactor = Config.Wrap<float>("Huntress", "ScaleFactor", "Set how much the range increases (in meters) each level. Recommended: 3 (but I'm not your dad)", 3f);
+            baseValue = Config.Wrap<float>("Huntress", "Base Value", "Base range (Original is 60), 100f);

```

Comment:
```suggestion
            baseValue = Config.Wrap<float>("Huntress", "Base Value", "Base range (Original is 60)", 100f);

```
Missed the closing quote